src/Symfony/Component/HttpFoundation/Session/Flash/FlashBagInterface.php

```diff
@@ -24,7 +24,7 @@
      * Adds a flash message for type.
      *
      * @param string $type
-     * @param string $message
+     * @param string|array $message
```

Comment:
Looking at https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/Session/Flash/FlashBag.php#L73 i dont think this is supported =/